src/components/full-calender/components/shared/register-form/reducer/usename.js

```diff
@@ -0,0 +1,14 @@
+const initialState = {
+    error: true,
+    value: "",
+    message: ""
+};
+
+export default (state = initialState, action)=>{
+    switch (action.type) {
+        case "USER_AVAILABILITY_DATA":return action.payload;
```

Comment:
it should be ...
case "RESET_USERNAME_DATA":
case "RESET_REGISTER_FORM":
return initialState;